src/core/config.py

```diff
@@ -0,0 +1,60 @@
+import glob
+import json
+import os
+from typing import Dict, List, Tuple
+
+from src.core import actions
+
+
+def per_file_process(
+    filename: str, ret_dict: dict, required_keys, action_key, filename_key
+) -> Tuple:
+    with open(filename, encoding="utf-8") as file:
+        data = json.load(file)
+        ret_dict.update(data)
+        if not ret_dict.get("enabled"):
+            # Only process enabled files
+            return None, None
+
+        action = ret_dict.get("action")
+        known_actions = actions.get_action_context_by_key(key=action_key)
+        if action not in known_actions:
+            assert False, f"CONFIG ERROR: Unknown action `{action}`."
```

Comment:
I noticed we do this assertion with a `CONFIG ERROR` message thing a few times in this function. Maybe this would be a good usecase for making our own `ConfigError(Exception)` and raising that instead?

Reply:
resolved in this commit:
https://github.com/mozilla/jira-bugzilla-integration/pull/1/commits/8219d4187d7cd8cde7dd014367fdf65aa6a9d98a